offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/create/SelectItemsView.groovy

```diff
@@ -302,7 +302,7 @@ class SelectItemsView extends VerticalLayout{
 
             //specify size of grid and layout
             grid.setWidthFull()
-            grid.setHeightMode(HeightMode.UNDEFINED)
+            grid.setHeightMode(HeightMode.ROW)
```

Comment:
Why did you choose to switch to `ROW` here?

Reply:
Because in the Future the grid will have a lot of packages and I thought it's better to fix the height so the page is not overflown with packages